.circleci/config.yml

```diff
@@ -44,4 +48,30 @@ jobs:
           command: |
             . venv/bin/activate
             bash <(curl -s https://codecov.io/bash)
-
+  build-py27:
+    docker:
+      - image: continuumio/miniconda:latest
+    steps:
+      - checkout
+      - restore_cache:
+          key: v1-dependency-cache-{{ checksum "setup.py" }}-{{ checksum "Makefile" }}-{{ checksum "requirements.txt" }}
```

Comment:
```suggestion
          key: v1-dependency-cache-{{ checksum "setup.py" }}-{{ checksum "Makefile" }}-{{ checksum "requirements.txt" }}-{{ checksum "requirements-dev.txt" }}
```

Reply:
We should also make this change for the Python3 dependency cache, but add a checksum for the new `requirements-docs.txt` as well. I don't think I can "suggest changes" on a line that wasn't edited by a PR though.